stonesoup/plotter.py

```diff
@@ -1368,3 +1367,815 @@ def update_animation(index: int, lines: List[Line2D], data_list: List[List[State
                     lines[i].set_data([],
                                       [])
         return lines
+
+
+class AnimatedPlotterly(_Plotter):
+    """
+    Class for a 2D animated plotter that uses Plotly graph objects rather than matplotlib.
+    This gives the user the ability to see how tracking works through time, while being
+    able to interact with tracks, truths, etc, in the same way that is enabled by
+    Plotly static plots.
+
+    Simplifies the process of plotting ground truths, measurements, clutter, and tracks.
+    Tracks can be plotted with uncertainty ellipses or particles if required. Legends
+    are automatically generated with each plot.
+
+    Parameters
+    ----------
+    timesteps: Collection
+        Collection of equally-spaced timesteps. Each animation frame is a timestep.
+    tail_length: float
+        Percentage of sim time for which previous values will still be displayed for.
+        Value can be between 0 and 1. Default is 0.3.
+    equal_size: bool
+        Makes x and y axes equal when figure is resized. Default is False.
+    sim_duration: int
+        Time taken to run animation (s). Default is 6
+    \\*\\*kwargs
```

Comment:
Sorry, I meant so that you can set the dimensions to a custom size. 

E.g if you are displaying two different plots of different data, but want to plot them on the same scales for comparison.